script.js

```diff
@@ -1,4 +1,107 @@
+var totalgames = 0;
+var userwins = 0;
+var tiedgames = 0;
+var computerwins = 0;
+var userName = "";
+var currentGameMode = "Waiting for User Name";
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  var myOutputValue = "";
+  if (currentGameMode == "Waiting for User Name") {
+    userName = input; // allow user to input userName
+    currentGameMode = "diceGame"; // toggle game mode to play game
+    myOutputValue = "Hello " + userName + ". Lets Play!";
+    return myOutputValue;
+  } else if (currentGameMode == "diceGame") {
+    if (input != "Scissors" || input != "Paper" || input != "Stone") {
```

Comment:
if (input != "Scissors" && input != "Paper" && input != "Stone") is the correct way